### PR TITLE
webide: remove usage of sdk docker image

### PR DESCRIPTION
**Pull request by bolek-wisniewski**
This PR builds the webide server image without relying on the sdk docker
image: https://github.com/DACH-NY/damlc-docker/blob/master/Dockerfile

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: web-ide/ide-server/Dockerfile [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
 USER sdk
 ENV CODESERVER_SHA256=4fe5b4d10d3048a5e5aa3e0772c09ece56241b91e76eeaa7404b4da292442881\
-    CODESERVER_VERSION=1.939-vsc1.33.1
-RUN mkdir -p /home/sdk/.local/share/code-server/extensions/ &&\
+    CODESERVER_VERSION=1.939-vsc1.33.1\
+    PATH="${PATH}:/home/sdk/.daml/bin"
+    
+RUN cd /home/sdk/ &&\
+    curl -Lo - "https://github.com/digital-asset/daml/releases/download/v${DAML_VERSION}/daml-sdk-${DAML_VERSION}-linux.tar.gz" | tar xzvf - --strip-components 1 &&\
+    ls -l &&\
```
Comment (neil-da) [inline web-ide/ide-server/Dockerfile]: Is this line necessary? Or just debug gunk that stuck around?

File: web-ide/ide-server/Dockerfile [hunk 2]
```diff
@@ -1,32 +1,43 @@
-ARG DAML_VERSION=0.12.16
+FROM openjdk:8-slim
 
-FROM "digitalasset/daml-sdk:${DAML_VERSION}-master"
+ARG DAML_VERSION=0.12.21
```
Comment (neil-da) [inline web-ide/ide-server/Dockerfile]: Can the bits we need to change regularly like the DAML version remain at the very top of the file?

Comment (bolek-wisniewski) [reply web-ide/ide-server/Dockerfile]: this is a dockerism where variables are undefined after FROM keyword.

File: web-ide/ide-server/Dockerfile (hunk 1 above)
Comment (bolek-wisniewski) [reply web-ide/ide-server/Dockerfile]: oops - yea debug

File: web-ide/ide-server/Dockerfile (hunk 2 above)
Comment (garyverhaegen-da) [reply web-ide/ide-server/Dockerfile]: Why does it need to be an ARG? Why not make it an ENV, and move the CODESERVER ones up here too?

Comment (bolek-wisniewski) [reply web-ide/ide-server/Dockerfile]: yea...I had moved them up...I swear I've been seeing outdated code and comments more often in github.

Saw your comment too late but having them set ENV variable implies to me that they are used inside the container, which they aren't. I don't really feel strongly about it though...

**Pull request merged**